components/camel-knative/camel-knative-component/src/main/docs/knative-component.adoc

```diff
@@ -137,13 +137,21 @@ RouteBuilder.addRoutes(context, b -> {
 <1> set the location of the `Knative Environment` file
 <2> expose knative service
 
-=== Using custom Knative Transport
+You can also use property based component configuration to set the `Knative Environemnt` configuration file.
 
-As today the component only support `http` as transport as it is the only supported protocol on Knative side but the transport is pluggable by implementing the following interface:
+.application.properties
+[source,properties]
+----
+camel.component.knative.environmentPath=classpath:knative.json
```

Comment:
Is this anyhow changing any previous behavior. I am not into details of the implementations, just wondering if by any chance we'll be introducing any breaking compatibility. I recall we gave the possibility to initialize environment from an environment variable as well, but I may be making confusion.

Reply:
no, this is not a new behavior and it is also working with `CAMEL_COMPONENT_KNATIVE_ENVIRONMENT_PATH` envVar